### PR TITLE
fix cleaner equation

### PR DESCRIPTION
**Pull request by aryantalekar20**
#### Checklist

 -[ x]Added description of change
  -[ x]Added file name matches File name guidelines
 -[ x] Added tests and example, test must pass
  -[ ]Added documentation so that the program is self-explanatory and educational - Doxygen guidelines
  -[x ]Relevant documentation/comments is changed or added
  -[x ]PR title follows semantic commit guidelines
  -[ x]Search previous suggestions before making a new one, as yours may be a duplicate.
  -[x ]I acknowledge that all my contributions will be made under the project's license.

Notes: 
- Changed the Djisktra Algorithm using a Graph data structure.

<a href="https://gitpod.io/#https://github.com/TheAlgorithms/C-Plus-Plus/pull/1927"></a>

### CODE REVIEW

**Review by Qannaf — APPROVED** [review]

File: greedy_algorithms/DJnew.cpp [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+ * @details
+ * Dijkstra's algorithm is an algorithm for finding the shortest paths between nodes in a graph using its neighbors
+ * description containing links, references,
+ * math equations, etc.
+ * @author [Aryan Talekar](https://github.com/aryantalekar20)
+ */
+
+#include <iostream>
+#include <vector>
+#include <set>
```
Comment (Panquesito7) [inline greedy_algorithms/DJnew.cpp:13]: Consider using `uint64_t` for non-negative values (or their appropriate size: `uint32_t`, `uint16_t`, `uint8_t`) or `int64_t` for negative values. Check other parts of the code ([reference](https://en.cppreference.com/w/cpp/types/integer)). **Let us know if you need any help here or in our [Discord server](https://discord.gg/c7MnfGFGa6).** 🙂

File: greedy_algorithms/DJnew.cpp [hunk 2]
```diff
@@ -0,0 +1,149 @@
+/**
+ * @file
+ * @brief Implementation of Djikstra's Algorithm
```
Comment (Panquesito7) [inline greedy_algorithms/DJnew.cpp:3]: Provide a Wikipedia link in Markdown format (if available/possible).
If there's no Wikipedia link, add another web source for algorithm explanation. 🙂

File: greedy_algorithms/DJnew.cpp [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+ * @brief Main function
+ * @param argc commandline argument count (ignored)
+ * @param argv commandline array of arguments (ignored)
+ * @returns 0 on exit
+ */
+int main() {
+    greedy_algorithms::test();  // run self-test implementations
+    // code here
+    return 0;
+}
```
Comment (Panquesito7) [inline greedy_algorithms/DJnew.cpp:149]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Nice work! 🚀
Please enable [GitHub Actions](https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions) in your repository of this fork in this link: https://github.com/aryantalekar20/C-Plus-Plus/actions

File: greedy_algorithms/DJnew.cpp (hunk 1 above)
Comment (aryantalekar20) [reply greedy_algorithms/DJnew.cpp:13]: Thank you! I just joined the discord server. I also enabled GitHub Actions.

**Pull request closed without merging**